src/pages/playerScreen/playerScreen.module.scss

```diff
@@ -0,0 +1,10 @@
+.playerWrapper {
```

Comment:
Player screen can be deleted as we planned to open player in fullscreen mode only